packages/v4/src/content/design-guidelines/styles/typography/typography.css

```diff
@@ -8,6 +8,8 @@
 
 #usage-guidelines ~ h3.ws-title {
   margin-top: 24px;
+  font-size: 18px;
+  fontWeight: 400;
```

Comment:
The "super hero text", "hero text", etc headings get a little lost for me. The other titles on the page use the title component classes to define the title styles. The hierarchy is 4xl (typography/h1), 2xl (our font family, usage guidelines), so this is what the h3's look like as an xl title.
<img width="1926" alt="Screen Shot 2022-03-02 at 10 16 25 AM" src="https://user-images.githubusercontent.com/35148959/156402619-7b63006e-fa5e-467b-8a7d-9d315695abc2.png">

Also fwiw, if you're defining font weight in a CSS file, it's written as `font-weight`. So in this case `fontWeight: 400;` isn't being applied.

Reply:
In addition to mcoker's note about changing `fontWeight` to `font-weight`, it looks like all the titles matching this css rule already have a font-weight of 400 so this isn't making any changes.